### PR TITLE
Toggle tags

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?166284

feature_flags

@czue @amsagoff 

enchantress @emord

### CODE REVIEW

File: corehq/feature_previews.py [hunk 1]
```diff
@@ -27,7 +28,7 @@ class FeaturePreview(StaticToggle):
     def __init__(self, slug, label, description, help_link=None, privilege=None, save_fn=None):
         self.privilege = privilege
         self.save_fn = save_fn
-        super(FeaturePreview, self).__init__(slug, label, description=description, help_link=help_link,
+        super(FeaturePreview, self).__init__(slug, label, TAG_PREVIEW, description=description, help_link=help_link,
```
Comment (dimagimon) [inline corehq/feature_previews.py]: E501 line too long (116 > 115 characters)

File: corehq/toggles.py [hunk 2]
```diff
@@ -1,14 +1,25 @@
+from collections import namedtuple
 from functools import wraps
 import hashlib
 from django.http import Http404
 import math
 from toggle.shortcuts import toggle_enabled, set_toggle
 
+Tag = namedtuple('Tag', 'name css_class')
+TAG_ONE_OFF = Tag(name='One-Off', css_class='important')
+TAG_EXPEREIMENTAL = Tag(name='Experimental', css_class='warning')
```
Comment (emord) [inline corehq/toggles.py]: Extra E in constant `TAG_EXPERIMENTAL`

File: corehq/toggles.py [hunk 3]
```diff
@@ -324,18 +373,21 @@ def toggles_dict(username=None, domain=None):
...
     'help_markdown',
     "Use markdown for the help text in the form builder",
+    TAG_UNKNOWN,
     [NAMESPACE_DOMAIN]
 )
 
 VELLUM_SAVE_TO_CASE = StaticToggle(
     'save_to_case',
     "Adds save to case as a question to the form builder",
+    TAG_UNKNOWN,
```
Comment (emord) [inline corehq/toggles.py]: Think this is experimental

**Comment by emord** [comment]
Looks good to me. Will let @amsagoff  comment on the feature's tags

**Comment by czue** [comment]
nice! did you match the colors with trello? if so that's a very nice touch.

**Comment by czue** [comment]
(imagine we can just merge and fix the tags later if needed - seems like we should fix the typo though)

**Comment by amsagoff** [comment]
: D : D : D

**Comment by emord** [comment]
@czue @snopoke  went ahead and fixed the type

**Comment by snopoke** [comment]
Thanks @emord.

I didn't match the trello colors but that's a good idea.

**Pull request merged**